elasticdl/python/ps/embedding_table.py

```diff
@@ -28,6 +31,7 @@ def __init__(self, name, dim=None, initializer=None, is_slot=False):
         """
         self.name = name
         self.dim = dim
+        self.initializer_pb = initializer
```

Comment:
Should check that `initializer` is a string because you add a function `to_embedding_info_pb` and it requires that `initializer` is a string.

I didn't take into consideration that `initializer` should be string if `is_slot=True`. In my impression, I passed in floating-point numbers in some unit tests... Or you can ignore this comment and I will fix this issue after your PR.

Reply:
I can cast `self.initializer_value` which has been renamed to string when I set it to `embedding_pb.initializer`.